include/posix/arpa/inet.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_POSIX_ARPA_INET_H_
+#define ZEPHYR_INCLUDE_POSIX_ARPA_INET_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <net/socket.h>
+
+static inline char *inet_ntop(sa_family_t family, const void *src, char *dst,
```

Comment:
Are you sure about this? These two functions don't seem to belong in this file: http://pubs.opengroup.org/onlinepubs/007908799/xns/arpainet.h.html

What SUS version are you targeting here?

Reply:
> What SUS version are you targeting here?

Targeting latest publicly available POSIX version: http://pubs.opengroup.org/onlinepubs/9699919799/

http://pubs.opengroup.org/onlinepubs/9699919799/functions/inet_ntop.html

Quick check: "man inet_ntop"

Reply:
I haven't been following the posix compatibility work carefully, but this PR makes me wonder if it's just piecemeal efforts to support individual applications, or a disciplined approach that's enabling functionality in a staged way.

I'm not asking for the entire POSIX API to be implemented in one go. That's a ridiculous strawman.

I *am* asking what the structure of the approach is, given that this PR seems to have problems in that regard.